apps/api/src/form/entity/form.entity.ts

```diff
@@ -1,12 +1,21 @@
-import { FormPayload } from '@ehpr/common';
-import { BaseEntity } from 'src/database/base.entity';
 import { Entity, Column } from 'typeorm';
+import { Exclude } from 'class-transformer';
+import { BaseEntity } from 'src/database/base.entity';
+import { PayloadDTO } from '@ehpr/common';
 
 @Entity('form')
 export class FormEntity extends BaseEntity {
   @Column('jsonb', { nullable: false })
-  payload!: FormPayload;
+  payload!: PayloadDTO;
+
+  @Column('varchar', { nullable: true })
+  confirmationId!: string;
+
+  @Exclude()
```

Comment:
Could you explain what the difference is between fields we want to exclude and fields we don't?

Reply:
This is for entity serialization. Only return the properties we want to send back as response.